snowpack/src/scan-imports.ts

```diff
@@ -274,6 +274,7 @@ export async function scanImportsFromFiles(
   loadedFiles: SnowpackSourceFile[],
   config: SnowpackConfig,
 ): Promise<InstallTarget[]> {
+  await initESModuleLexer;
```

Comment:
@FredKSchott this was added because `parseFileForInstallTargets()` below calls `parse()`, and that may have been called before `await init`. This might not change anything, but this should help

Reply:
yup, safe to remove the original `await initESModuleLexer;` now that this is here. 

I mean, the whole idea of "this only works if you call this other thing implicitly" is kind of gross, but I like this better than converting the entire function stack to async, at least for now